tests/integration/plugins/test_application_plugin.py

```diff
@@ -145,7 +145,7 @@ def test_application_plugin_missing_stuff(new_dir, partitions):
         )
 
     assert raised.value.part_name == "foo"
-    assert raised.value.message == "- field 'app-stuff' is required"
+    assert raised.value.message == "- Field required in field 'app-stuff'"
```

Comment:
I think we need to handle this better in `from_validation_error()`, because now it looks awkward "Field required in field"

Reply:
Indeed. Perhaps `Field required under 'app-stuff'` will sound better?

Reply:
well the previous message `field 'app-stuff' is required` looked fine to me!

Reply:
Sorry, you're right. I thought the missing field was a field under "app-stuff" and not "app-stuff" itself.